src/styles/index.scss

```diff
@@ -1,3 +1,226 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin: 0;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
```

Comment:
Checklist item: 'Don't style using ids'. The '.header' class should not set the font-family and font-size since these are inherited from the 'body' tag. Moreover, the font-family is already set in the 'body' tag and should not be repeated unless there is a need to override it.